x-pack/platform/plugins/shared/maps/public/react_embeddable/map_react_embeddable.tsx

```diff
@@ -206,13 +207,17 @@ export const mapEmbeddableFactory: ReactEmbeddableFactory<
         );
 
         useEffect(() => {
+          mapSettings$.next({
```

Comment:
Re-setting `backgroundColor` removes user selected background color.

To view problem
1) create a new map
2) remove baselayer
3) add EMS boundaries world country layer.
4) Clicking "settings" in map top nave and provide a background color.
5) add map to dashboard, save dashboard.
6) re-open dashboard. Notice how custom background color is removed.

Reply:
Oh yeah I'll remove this bit of code. I opted to make the default background color transparent instead of having to pass an EUI color into the default map settings object, which also fixes the white flash that you see on some maps in dark mode.